### ISSUE TITLE
Use CSS for flags

### ISSUE THREAD

**Issue by derchrisuk** [webSPELL/webSPELL#236]
We should implement the use of CSS for flags:

http://lipis.github.io/flag-icon-css/

Easy to scale.

**Comment by Phhere** [comment]
I do not think that this is useful because not every browser renders these correct

**Comment by derchrisuk** [comment]
All major browsers I tried with worked ok.
